stacks_queues/stack.py

```diff
@@ -12,7 +12,7 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_first(element)
 
     def pop(self):
         """ Removes an element from the top
```

Comment:
✨remove_first() returns the value already, so using `get_first()` to store the value and then return it isn't necessary!